nixos/services/varnish/default.nix

```diff
@@ -190,9 +189,6 @@ in
     environment.etc."varnish/startup.sh".source = startupscript;
 
     systemd.services.varnish = {
-      preStart = lib.mkBefore ''
-        rm -rf ${vcfg.stateDir}
-      '';
```

Comment:
@ctheune Or is this still required for cleanup of upgrading installations?

Reply:
I don't like that this happens in here. Is this required to make a test green?